.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -30,6 +30,29 @@ body:
       description: A clear description of the bug you have found. Please include relevant information and resources (for example the steps to reproduce the bug)
     validations:
       required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to Reproduce
+      description:  To help us recreate the bug, provide a numbered list of the exact steps taken to trigger the buggy behavior.
+      value:  | 
+        <!--- 
```

Comment:
Thank you for the improvements 👍  What about not having it commented out?

Reply:
I think then they can edit it and it will display

From other experience I found people leave it commented out and wonder why it disappears

Reply:
Oh okay. In that case shall I just remove the comment for both and simply leave it?

Reply:
Without any comment, it would look like this.. when creating a issue (bug) 

![image](https://github.com/EddieHubCommunity/BioDrop/assets/57263951/a7cf04bf-404d-41e4-8c8b-7c276cf4f0e2)

> I think then they can edit it and it will display
> 
> From other experience I found people leave it commented out and wonder why it disappears

Now It makes perfect sense to me. We should go without commenting anything.

Reply:
Looks good 👍